### ISSUE TITLE
browserify should be npm as the title is "Package Managers"

### ISSUE THREAD

**Issue by stevemao** [sorrycc/awesome-javascript#139]
browserify - Browser-side require() the node.js way.
`browserify` should probably be `npm`. `npm` is the package manager and `browserify` is the build tool.
I would reword this to
npm - when built with browserify, you can require() the node.js way in browser.

**Comment by huei90** [comment]
How about this

Browserify lets you require('modules') in the browser by bundling up all of your dependencies.

Copy from http://browserify.org

**Comment by stevemao** [comment]
Sure. I'm not worried about the wording but the point is `npm` should be the package manager :smile:

**Comment by huei90** [comment]
No worry.

https://github.com/sorrycc/awesome-javascript/commit/20580959be3af2c075d5563deb702704eccc734a

**Comment by stevemao** [comment]
Sorry @huei90 , I meant the name of package manager should be `npm` because `browserify` is just a build tool for packages that are commonJS style. Most packages on `npm` are commonJS style so you could relate them. 

Or even better, I think browserify should be under category "Loaders" because its a tool to bundle commonJS modules (similar to webpack which is already under the category). Then add `npm` under category "Package Managers" because `npm` is a package manager of JavaScript (including browser-side). There are many browser-side packages (which are not even commonJS style) such as angular.js

**Comment by huei90** [comment]
hmmm I confuse now.
